adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMAdapterVungleUtils.m

```diff
@@ -26,7 +26,10 @@
 }
 
 const CGSize kVNGBannerShortSize = {300, 50};
-GADAdSize GADMAdapterVungleAdSizeForAdSize(GADAdSize adSize) {
+GADAdSize GADMAdapterVungleAdSizeForAdSize(GADAdSize adSize, NSString *_Nonnull placementId) {
```

Comment:
Does this method not used any more?

Reply:
yes not used. but shiv wanted us to leave it.